sdks/java/core/src/main/resources/org/apache/beam/sdk/sdk.properties

```diff
@@ -16,7 +16,7 @@
 #
 # SDK source version
 
-version=${pom.version}
+version=@pom.version@
```

Comment:
<!--new_thread; commit:e5a7a496d8ce72745e7d44fe3a48e79091b38336; resolved:0-->
Ditto here that we might not make this change unless maven and gradle both support the same syntax. I'm sure we could find a way. But maybe this just works for both?

Reply:
Are there tests of what this is expected to be?